.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Linux tests
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  linux:
+    name: Linux py${{ matrix.PYTHON_VERSION }} tests
+    runs-on: ubuntu-latest
```

Comment:
baby steps, I know, but windows (and to a lesser extent, osx) is where the real problems usually occur.

Reply:
OSX build machines are "expensive" So maybe on OSX we should only run  oldest and newest?

Reply:
too true, everything has a cost! i think as long as there is _some_ coverage on all the platforms, it's solid. another thing i've done before, when the wall-clock time isn't as important as the overall resource footprint, is do strongest and weakest indicator builds by themselves (in this case, I guess that would be windows 3.5 and and linux 3.8?) and then open up the matrix, but still set a max concurrency.